core/lib/dal/src/consensus/proto/mod.proto

```diff
@@ -13,10 +13,30 @@ message Payload {
   optional uint64 fair_pubdata_price = 11; // required since 1.4.1; gwei
   optional uint32 virtual_blocks = 6; // required
   optional bytes operator_address = 7; // required; H160
+  // Set for protocol_version < 25.
   repeated Transaction transactions = 8;
+  // Set for protocol_version >= 25.
+  repeated TransactionV25 transactions_v25 = 12;
   optional bool last_in_batch = 10; // required
 }
 
+message L1Transaction {
```

Comment:
naming: L1Transaction**V25** / L2Transaction**V25** ?

Reply:
ack, however this is supposed to be new standard encoding and it is the role of the top level message to indicate which protocol version it should be used with, so I'd prefer to avoid spamming V25 suffix too much.